### DISCUSSION TITLE
Multi-tenancy in v3

### DISCUSSION THREAD

**Discussion by rafacompu** [thedevdojo/wave#147] [Ideas]
Hello all, 

Has anyone tinker with the idea to make the project multi tenant?

I've had projects in the past using https://tenancyforlaravel.com/ and it works great.

Tried to add this package to the base v3 but I've gotten nowhere.

**Comment by bobbyiliev** [comment]
Hey!

I've not tested this either, but as Wave v3 uses Fillament, maybe this here will work:

> https://filamentphp.com/docs/3.x/panels/tenancy

Though I've not tried it myself so not 100% sure.

**Reply by rafacompu** [reply]
I don't think that's going to do a sub-domain tenancy. It seems more like to split the app into teams?

**Reply by bobbyiliev** [reply]
I think that subdomain routing is not specific to that tenancy for Laravel package, but something that comes out of the box with a plain Laravel app as well:

> https://laravel.com/docs/11.x/routing#route-group-subdomain-routing

On the Tenancy for Laravel package topic, it seems like someone was able to get this to work with Filament:

> https://github.com/filamentphp/filament/discussions/2058

Though again, I've not tested this so can't really confirm if this will all work out fo the box without some major changes.

**Comment by krecco** [comment]
Has anybody had success using third party libraries?
